org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_detail.scss

```diff
@@ -232,7 +232,11 @@
 
       .crm-button-type-cancel,
       .crm-button_qf_Address_1_cancel,
-      .crm-button_qf_Address_2_cancel {
+      .crm-button_qf_Address_2_cancel,
```

Comment:
Could this be simply
```css
[class*="crm-button_qf_Address_"] {

}
```
?

This would work with `.crm-button_qf_Address_1_cancel`, `.crm-button_qf_Address_3_cancel`, etc..but also with `.crm-button_qf_Address_2_something_else` so be careful that it doesn't have unintended side effects